Dart/src/com/jetbrains/lang/dart/assists/AssistUtils.java

```diff
@@ -48,13 +49,25 @@
 
 public class AssistUtils {
   public static void applyFileEdit(@NotNull final SourceFileEdit fileEdit) {
+    final Set<String> excludedIds = Sets.newHashSet();
```

Comment:
here and below: IMO `Collections.emptySet()` would better reflect that this map is not going to be updated inside and we really do not want to exclude anything.